pkg/validator/validator.go

```diff
@@ -230,6 +230,22 @@ func Validate(data ValidatorData) (Report, error) {
 	}, nil
 }
 
+func GetKubeletConfigurationsFromTASEnabledNodes(ctx context.Context, cli client.Client) (map[string]*kubeletconfigv1beta1.KubeletConfiguration, error) {
```

Comment:
this is calling for some refactoring, but can be postponed

Reply:
If we do not need to check KubeletConfigurations in the end we could avoid this function completely